### DISCUSSION TITLE
Mutant Chicken Island

### DISCUSSION THREAD

**Discussion by Blasta-alasta** [sunflower-land/sunflower-land#2757] [Ideas]
### MUTANT CHICKEN ISLAND - BURN MUTANTS CHICKENS FOR MYSTERY CHESTS?

**Lore:**

_The mad scientist Dr. ChickenX and his hunchback Henchman Deekol have found a mysterious island with volcano at its centre. At the base of this volcano is an ancient temple dedicated to what looks like Chicken Gods, belonging to a Bumpkin civilisation long forgotten._

_Dr ChickenX has been excavated the site, and has discovered fossilised Chicken DNA. By conducting mad experiments with this he able to produce beyond belief wonder, adept sometimes some of his failure turn out to be monsters! The mad Dr has sent his henchman Deekol in search for volunteers willing to sacrifice valuable resources in exchange for one of these special wonders._

_It is the mad Doctors experiments that render some of the most amazing mutant chickens the Isles have ever seen!_

**Idea:**

The three main Mutants Chickens have an unlimited supply, and no burning mechanism.

I had an idea of an island where players can burn Mutant Chickens to craft _MYSTERY “CLUCK LUCK” BOXES_. Similar to the idea with the Genie Lamp, the Box once activated will give the player a chicken related reward.

Different rewards would have different drop rates similar to the mechanics deployed in TI.
Common items could be rare decorations, and rarer items could have different boosts depending on their rarity.

This is a feature/island that could have no time limit. Restrictions could be put in place to limit the amount of Boxes crafted by a player (e.g. one box per 3 days). And then it gives a constant burn for the below items.

_**Proposal: Mystery “CLUCK LUCK BOXES” Boxes**_

**BASIC CHICKY CHEST - Cluck for Luck”**
_Requirements:_
	50 	x 	Eggs
		10	x	Gold
		1 	x 	Block Buck

**DELUXE CHICKY CHEST - “More Luck for your Buck”**  (Removes Common Prizes)
_Requirements:_
100 	x 	Eggs
		10	x	Gold
		1	x 	Mutant Chicken (any one)
		1 	x 	Block Buck

![Screenshot 2023-07-31 at 9 20 02 PM](https://github.com/sunflower-land/sunflower-land/assets/102267293/b22d3386-5197-4885-9d7a-fa93ed6ce516)

The idea to keep the Island continuously running would be to make the percentages for the rarer rewards quite low.

![Screenshot 2023-07-31 at 9 19 53 PM](https://github.com/sunflower-land/sunflower-land/assets/102267293/c5afc87f-39a4-4617-ac3b-4518679b8d19)

What do you guys think?

**Comment by paduv** [comment]
I like the idea, but i think boosts should be only chicken-related, with maybe very few exceptions.

The exceptions should be logical, so still chicken-related in a way
ex:
- boost on wheat (crops, time, drop chance, ..)
- boosts on cakes (you've got nice ideas there) & other egg-related productions
- anything if we can find a legitimate link with chickens.. :)

Other drops could be added to egg production, giving more utility to chicken & related items:
- sfl % drop with egg harvesting
- golden egg  :p  (% drop of gold) + why not iron egg, stone egg, wood egg, ..
- ...
